ArduCopter/mode.cpp

```diff
@@ -422,6 +422,16 @@ void Copter::update_flight_mode()
     surface_tracking.invalidate_for_logging();  // invalidate surface tracking alt, flight mode will set to true if used
 
     flightmode->run();
+
+#if HAL_MOUNT_ENABLED
+    {
+        const bool is_landing = flightmode->is_landing();
```

Comment:
This is fine.  An alternative implementation (which I suspect you also thought of) was to pass the is_landing() state into the mount and let it decide if the state has changed.  It's no big deal but done that way we might move a bit more code into AP_Mount if we choose to implement this for Plane as well.

Reply:
Yep, I certainly did consider that.  But then I thought that other libraries might *also* want to know when we've just transitioned into landing.  Kind if figured we'd see which way things went.

It also briefly crossed my mind to use Notify events but quickly discarded that concept.